src/widget/wlabel.h

```diff
@@ -46,6 +58,7 @@ class WLabel : public QLabel, public WBaseWidget {
     QString m_longText;
     Qt::TextElideMode m_elideMode;
     double m_scaleFactor;
+    int m_highlight;
```

Comment:
Shouldn't this be a boolean?

Reply:
it's connected to a CO and that can have N states, like [Shoutcast],status.

btw, the entire code is copied over from wwidgetgroup.

Reply:
ok